### PR TITLE
Rad2deg paddle frontend

### LINKED ISSUE #17644: rad2deg

### PR DESCRIPTION
**Pull request by selvaraj-sembulingam**
Close #17644

### CODE REVIEW

**Comment by selvaraj-sembulingam** [comment]
## Frontend Task Checklist
#### IMPORTANT NOTICE 🚨:
The [Ivy Docs](https://unify.ai/docs/ivy/) represent the ground truth for the task descriptions and this checklist should only be used as a supplementary item to aid with the review process.

#### LEGEND 🗺:
- ❌ :  Check item is not completed. 
- ✅ :  Check item is ready for review.
- 🆘 :  Stuck/Doubting implementation (PR author should add comments explaining why).
- ⏩ :  Check is not applicable to function (skip).
- 🆗 :  Check item is implemented and does not require any edits.

#### CHECKS 📑:
1. - [x] ✅: The function/method definition is not missing any of the original arguments.
2. - [x] ⏩: In case the function/method to be implemented is an alias of an existing function/method:
       1. - [x] ⏩: It is being declared as such by setting `fun1 = fun2`, rather than being re-implemented from scratch.
       2. - [x] ⏩: The alias is added to the existing function/method's test in the `aliases` parameter of `handle_frontend_test`/`handle_frontend_method`.
3. - [x] ✅: The naming of the function/method and its arguments exactly matches the original.
4. - [x] ✅: No defined argument is being ignored in the function/method's implementation.
5. - [x] ⏩: In special cases where an argument's implementation should be pending due to an incomplete superset of an ivy function:
       1. - [x] ⏩: A descriptive comment has been left under the `Implement superset behavior` ToDo list in https://github.com/unifyai/ivy/issues/6406.
       2. - [x] ⏩: A ToDo comment has been added prompting to pass the frontend argument to the ivy function whose behavior is to be extended.
6. - [x] ✅: In case a frontend function is being added:
       1. - [x] ⏩: It is a composition of ivy functions.
       2. - [x] ⏩: In case the needed composition is long (using numerous ivy functions), a `Missing Function Suggestion` issue has been opened to suggest a new ivy function should be added to shorten the frontend implementation. 
       3. - [x] ✅: `@to_ivy_arrays_and_back` has been added to the function.
7. - [x] ✅: In case a frontend method is being added:
       1. - [x] ⏩: It is composed of existing frontend functions or methods. 
       2. - [x] ⏩: If a required frontend function has not yet been added, the method may be implemented as a composition of ivy functions, making sure that:
              - [x] ✅: `@to_ivy_arrays_and_back` has been added to the method.
              - [x] ⏩: A ToDo comment has been made prompting to remove the decorator and update the implementation as soon as the missing function has been added.
8. - [x] ✅: The function/method's test has been added (except in the alias case mentioned in <2>):
       1. - [x] ✅: All supported arguments are being generated in `handle_frontend_test`/`handle_frontend_method` and passed to `test_frontend_function`/`test_frontend_method`. 
       2. - [x] ✅: The argument generation covers all possible supported values. Array sizes, dimensions, and axes adhere to the full supported set of the original function/method.
       3. - [x] ✅: The `available_dtypes` parameter passed to the helper generating the function/method's input array is set to `helpers.get_dtypes("valid")`. If there are unsupported dtypes that cause the test to fail, they should be handled by adding `@with_supported_dtypes`/`@with_unsupported_dtype` to the function/method.
9. - [x] ✅: The PR is not introducing any test failures.
       1. - [x] ✅: The lint checks are passing.
       2. - [x] ✅: The implemented test is passing for all backends.
10. - [x] ✅: The PR `closes` a `Sub Task` issue linked to one of the open frontend ToDo lists.
11. - [x] ✅: The function/method and its test have been added to the correct `.py` files corresponding to the addressed ToDo list.
12. - [x] ✅: The PR only contains changes relevant to the addressed subtask.

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -276,3 +276,7 @@ def greater_than(self, y, name=None):
     @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
+
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
```
Comment (aparajith21) [inline ivy/functional/frontends/paddle/tensor/tensor.py:280]: Thanks for contributing! LGTM, minor thing: Could this be modified to 2.5.0

Comment (selvaraj-sembulingam) [reply ivy/functional/frontends/paddle/tensor/tensor.py]: Done

**Comment by aparajith21** [comment]
LGTM! Thanks for contributing!

**Pull request merged**